index.js

```diff
@@ -7,34 +8,25 @@ const restrictedRoute = require('./routers/restrictedRouter.js');
 
 const server = express();
 
-server.use(
-    session({
-      name: 'notsession', // default is connect.sid
-      secret: 'nobody tosses a dwarf!',
-      cookie: {
-        maxAge: 1 * 24 * 60 * 60 * 1000,
-        secure: true, // only set cookies over https. Server will not send back a cookie over http.
-      }, // 1 day in milliseconds
-      httpOnly: true, // don't let JS code access cookies. Browser extensions run JS code on your browser!
-      resave: false,
-      saveUninitialized: false,
-    }));
-server.use(express.json());
+const sessionConfig = 
+    {
+        name: 'newApp', 
+        secret: 'wowowowowowayaya',
+        cookie: {
+            maxAge: 1000 * 60 * 10,
```

Comment:
How much time is this exactly? I think 10 minutes, but I could easily be wrong (so some comment or something slightly more explicit here would be nice).